Melonie's Solution/DB Challenge/sql.sql

```diff
@@ -0,0 +1,5 @@
+CREATE DATABASE Startup;
+USE Startup;
+CREATE TABLE `Order`(RecordID INT(4) PRIMARY KEY, `Name` VARCHAR(60), CellPhone VARCHAR(15), WorkPhone VARCHAR(15), Email VARCHAR(30), Address VARCHAR(60), BasicWidgetOrder INT(3), AdvancedWidgetOrder INT(3), ProtectionPlan BOOLEAN DEFAULT false);
```

Comment:
You seem to have maintained the same table structure as what the original NoSQL document had prior. Given that you're "transferring" data from NoSQL to a relational database, is there any further action you would take regarding the organization of the data before it gets placed into the "import" table? Additionally, would you consider the table structure you have currently as normalized?